sql/src/main/java/io/crate/metadata/ReferenceInfo.java

```diff
@@ -88,26 +89,28 @@ public ReferenceInfo() {
     public ReferenceInfo(ReferenceIdent ident,
                          RowGranularity granularity,
                          DataType type) {
-        this(ident, granularity, type, ColumnPolicy.DYNAMIC, IndexType.NOT_ANALYZED);
+        this(ident, granularity, type, ColumnPolicy.DYNAMIC, IndexType.NOT_ANALYZED, false);
     }
 
     public ReferenceInfo(ReferenceIdent ident,
                          RowGranularity granularity,
                          DataType type,
                          ColumnPolicy columnPolicy,
-                         IndexType indexType) {
+                         IndexType indexType,
+                         boolean isNotNull) {
         this.ident = ident;
         this.type = type;
         this.granularity = granularity;
         this.columnPolicy = columnPolicy;
         this.indexType = indexType;
+        this.isNotNull = isNotNull;
```

Comment:
I think it would be easier to read if this was just `nullable` instead of being in a negated form